cmd/bee/cmd/start.go

```diff
@@ -194,7 +196,8 @@ Welcome to the Swarm.... Bzzz Bzzzz Bzzzz
 	cmd.Flags().String(optionNameTracingServiceName, "bee", "service name identifier for tracing")
 	cmd.Flags().String(optionNameVerbosity, "info", "log verbosity level 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=trace")
 	cmd.Flags().String(optionWelcomeMessage, "", "send a welcome message string during handshakes")
-	cmd.Flags().Uint64(optionNameDisconnectThreshold, 100000000000, "threshold in BZZ until which you allow peers to be indebted before disconnecting")
+	cmd.Flags().Uint64(optionNamePaymentThreshold, 100000, "threshold in BZZ where you expect to get paid from your peers")
+	cmd.Flags().Uint64(optionNamePaymentTolerance, 10000, "excess debt above payment threshold in BZZ where you disconnect from your peer")
```

Comment:
Suggest to make comment that this should be maximally 1/2 * PaymentThreshold. Or add a hard check elsewhere

Reply:
Added a check during startup

Reply:
@janos not sure if this is the right place in the code for this kind of check: https://github.com/ethersphere/bee/pull/465/files#diff-dabbd93cd3d162371b93260a9f99c90cR103. What do you think? Should this be here, in `NewBee` or somewhere else?

Reply:
Sorry for a late response. I think that this is a logic that belongs to the accounting. Since both values are passed to the accounting constructor it should be validated there. It would contain the accounting logic into one package, not to leak to cmd.

Reply:
I moved this check into the accounting constructor which can now return an error. I also added a test for this as this is now testable.